tests/queries/0_stateless/01591_window_functions.sql

```diff
@@ -545,3 +545,7 @@ select count() over (w) from numbers(1) window w as ();
 
 -- nonexistent parent window
 select count() over (w2 rows unbounded preceding); -- { serverError 36 }
+
+-- ntile
```

Comment:
I'd suggest create separate test file for this function and cover more cases (different arg types, corner cases with illegal arguments and unsupported frames, etc)

Reply:
Result with Float argument unexpected:
```sql
select
	number % 2 as a,
	number as b,
	ntile(2 :: Float64) over (partition by a order by b)  as ntile
from numbers(10);


┌─a─┬─b─┬─ntile─┐
│ 0 │ 0 │     1 │
│ 0 │ 2 │     2 │
│ 0 │ 4 │     3 │
│ 0 │ 6 │     4 │
│ 0 │ 8 │     5 │
│ 1 │ 1 │     1 │
│ 1 │ 3 │     2 │
│ 1 │ 5 │     3 │
│ 1 │ 7 │     4 │
│ 1 │ 9 │     5 │
└───┴───┴───────┘

```

Same for string: `ntile('2')`

Reply:
Non constant argument probably should be forbidden also
```sql
select number%2, number, ntile(number + 2) over (partition by number % 2 order by number)  as ntile from numbers(10);
```